File: src/commonMain/kotlin/mahjongutils/models/Wind.kt
package mahjongutils.models

enum class Wind(val tile: Tile) {
    East(Tile.get(TileType.Z, 1)),
    South(Tile.get(TileType.Z, 2)),
    West(Tile.get(TileType.Z, 3)),
    North(Tile.get(TileType.Z, 4)),
}

File: python-lib/mahjong_utils/models/wind.py
from enum import Enum

from mahjong_utils.models.tile import Tile
from mahjong_utils.models.tile_type import TileType


class Wind(int, Enum):
    """
    风
    """

    east = 0
    south = 1
    west = 2
    north = 3

    @property
    def tile(self) -> Tile:
        return Tile.by_type_and_num(TileType.Z, self + 1)


__all__ = ("Wind",)
